apollo-router/src/spec/query/change.rs

```diff
@@ -132,158 +214,268 @@ impl<'a> QueryHashVisitor<'a> {
                 b.hash(self);
```

Comment:
I think it worth to add `^` to prefixes here, even just for consistency.
`^enum`
`^variable`
`^string`
etc

Reply:
It will also make debugging easier.
The last time I debugged this code, I added `println` to print all hashed values on separate lines.
So having `^` would make it easier to navigate

Reply:
fixed in 350fde0503c14f91eebf89f67f95f8f3b308c09a